client/components/Settings.tsx

```diff
@@ -80,34 +81,38 @@ const Settings = () => {
           <Text style={styles.buttonText}>Update name</Text>
         </Pressable>
 
-        <Text style={styles.title}>Choose reading level: {selectedProfile?.readingLevel}</Text>
-        <Picker
-          selectedValue={selectedProfile?.readingLevel}
-          style={styles.picker}
-          onValueChange={handleReadingLevelChange}
-        >
-        {Object.keys(readingLevelOptions).map((level) => (
-          <Picker.Item key={level} label={`${level}`} value={level} />  
-        ))}
-        </Picker>
-
-        <Pressable style={styles.button} onPress={handleDeleteProfile}>
-          <Text style={styles.buttonText}>Delete profile</Text>
+        <Text style={styles.title}>Update Reading Level</Text>
+        <View style={styles.pickerContainer}>
+          <Picker
+            selectedValue={selectedProfile?.readingLevel}
+            style={styles.picker}
+            onValueChange={handleReadingLevelChange}
+            dropdownIconColor='#ffffff'
+          >
+            {Object.keys(readingLevelOptions).map((level) => (
+              <Picker.Item key={level} label={`${level}`} value={level} />  
+            ))}
+          </Picker>
+        </View>
+
+        <Pressable onPress={handleDeleteProfile}>
+          <Ionicons name="trash-outline" size={24} color="#FFFFFF" />
         </Pressable>
 
         <Text style={styles.title}>Email: {user?.email}</Text>
 
-        <Pressable style={styles.button} onPress={handleNewProfile}>
-          <Text style={styles.buttonText}>Create new profile</Text>
-        </Pressable>
-
+      
+        <BlueButton title="+ New Profile" onPress={handleNewProfile}/>
         <OrangeButton title="Logout" onPress={handleLogout}/>
         
       </View>
     </>
   )
 }
 
+
+
+
 const styles = StyleSheet.create({
```

Comment:
switch to tailwind?

Reply:
yes, i just pushed early so that Regina could access. I still have a sryle container now, I have pushed for now so its ready for presentation but will review this afternoon